pkg/cache/tas_flavor_snapshot.go

```diff
@@ -267,6 +267,13 @@ func (s *TASFlavorSnapshot) addTASUsage(domainID utiltas.TopologyDomainID, usage
 }
 
 func (s *TASFlavorSnapshot) removeTASUsage(domainID utiltas.TopologyDomainID, usage resources.Requests) {
+	if s.leaves[domainID] == nil {
+		// this can happen if there is an admitted workload for which the
+		// backing node was deleted or is no longer Ready (so the addCapacity
+		// function was not called).
+		s.log.Info("skip removing TAS usage in domain", "domain", domainID, "usage", usage)
```

Comment:
```suggestion
		s.log.V(3).Info("skip removing TAS usage in domain", "domain", domainID, "usage", usage)
```

This is actually a debug log.

Reply:
I would argue it is an exception situation when nodes are deleted while the workloads are running. 

Also, I think it makes sense to be consistent with the analogus check [here](https://github.com/kubernetes-sigs/kueue/blob/17e6c6e20ebdd8582ab284f018aee93dd4ef7187/pkg/cache/tas_flavor_snapshot.go#L256-L262)